src/main/java/com/fvv/bookstore/util/DateUtil.java

```diff
@@ -47,4 +47,13 @@ public static String dateToString(final Date date) {
 			return null;
 		}		
 	}
+	
+	/**
+	 * Gets the current date and time in a specific format.
+	 * 
+	 * @return the formated date.
+	 */
+	public static String getCurrentDateTime() {
+		return LocalDateTime.now().format(DateTimeFormatter.ofPattern("dd-MM-yyyy HH:mm:ss"));
```

Comment:
put the mask in a constant in the Constant class

Reply:
ok